pkg/util/proxyutil/proxyutil.go

```diff
@@ -130,3 +132,32 @@ func ApplyForwardIDHeader(req *http.Request, user identity.Requester) {
 		req.Header.Set(IDHeaderName, token)
 	}
 }
+
+func ApplyTeamHeaders(req *http.Request, ds *datasources.DataSource, teams []int64) {
+	if ds.JsonData != nil && ds.JsonData.Get("teamHeaders").MustBool(false) {
```

Comment:
`ds.JsonData.Get("teamHeaders").MustBool(false)` is not necessary. We can also use early return to avoid nested if expression
```suggestion
	if ds.JsonData != nil {
```

Reply:
resolved